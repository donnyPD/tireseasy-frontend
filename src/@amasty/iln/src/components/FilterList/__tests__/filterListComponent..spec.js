import React from 'react';
import { createTestInstance } from '@magento/peregrine';
import FilterListComponent from '../filterListComponent';

jest.mock('../filterList', () => 'FilterList');
jest.mock('../filterTypeConfig', () => {
  return {
    filterTypeConfig: 'filterTypeConfig'
  };
});

test('renders default filter list', () => {
  const props = {
    filterBlockSettings: {
      display_mode_label: undefined
    }
  };

  const tree = createTestInstance(<FilterListComponent {...props} />);
  expect(tree.toJSON()).toMatchSnapshot();
});

test('renders from config', () => {
  const props = {
    filterBlockSettings: {
      display_mode_label: 'Slider'
    }
  };

  const tree = createTestInstance(<FilterListComponent {...props} />);
  expect(tree.toJSON()).toMatchSnapshot();
});
