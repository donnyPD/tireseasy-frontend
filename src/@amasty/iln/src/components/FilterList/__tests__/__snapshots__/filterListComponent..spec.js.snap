// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default filter list 1`] = `
<FilterList
  filterBlockSettings={
    Object {
      "display_mode_label": undefined,
    }
  }
/>
`;

exports[`renders from config 1`] = `
<FilterList
  filterBlockSettings={
    Object {
      "display_mode_label": "Slider",
    }
  }
/>
`;
